content/Canvas/canvasrenderingcontext2d.jsdoc

```diff
@@ -568,13 +568,37 @@ Draw the path with %%#fill|**fill()**%% or
   var canvas = document.getElementById('canvas');
```

Comment:
I think it would make more sense to replace the lineWidth example instead of beginPath.

Reply:
ah, of course! *facepalm*